### DISCUSSION TITLE
Apple M1 support

### DISCUSSION THREAD

**Discussion by drakkan** [drakkan/sftpgo#385] [General]
Hi,

I just added apple M1 support to the [CI](https://github.com/drakkan/sftpgo/commit/b11865f97141d5b58823a391f4a99cc20bd5204b). I have no way to test the produced binaries.

I think very few users use SFTPGo on macOS however if anyone has an ARM64 based Mac I will appreciate if he/she can download the `sftpgo-macos-latest-go-1.16` build artifact from this [page](https://github.com/drakkan/sftpgo/actions/runs/768535743) uncompress it and test if it works using the following commands:

```
chmod 755 sftpgo_arm64
./sftpgo_arm64 -v
./sftpgo_arm64 serve -l ""
```

thank you

**Comment by sergejostir** [comment]
Just tested on Mac mini M1 without a problem. I firstly ran `./sftpgo_arm64 serve -l ""`, but then used the portable mode to browse the files a bit with a client. Let me know, if you need anything else.

**Reply by drakkan** [reply]
Thank you!!!

**Reply by sergejostir** [reply]
I played around a few hours today using this build and haven't noticed any problems. Works just like other builds.

**Reply by drakkan** [reply]
> I played around a few hours today using this build and haven't noticed any problems. Works just like other builds.

Thank you for your testing, much appreciated
